src/Misc/layoutroot/run-helper.sh.template

```diff
@@ -31,9 +31,18 @@ elif [[ $returnCode == 2 ]]; then
     "$DIR"/safe_sleep.sh 5
     exit 2
 elif [[ $returnCode == 3 ]]; then
```

Comment:
do we need to do something to the `.cmd` version?

Reply:
We considered it, but this issue is unlikely to come up with windows machines as it usually happens on containers with unreliable IO, and these containers are overwhelmingly not running windows.